index.js

```diff
@@ -8,7 +8,6 @@ var AbstractLevelDOWN = require('abstract-leveldown').AbstractLevelDOWN
 var inherits = require('inherits')
 var Iterator = require('./iterator')
 var mixedToBuffer = require('./util/mixed-to-buffer')
-var isDataCloneError = require('./util/is-data-clone-error')
```

Comment:
Maybe this file can be removed and the function moved into `test/structured-clone-test.js`?